en/docs/administer/configure-enterprise-login.md

```diff
@@ -107,31 +107,27 @@ Before you proceed with the configuration, make sure you complete the following:
     3. Click **New Attribute** and select the **Groups** attribute.
     4. Click **Save** and then click **Save Changes**.
 
-### Step 2: Map Asgardeo groups to Choreo roles in the Choreo Console 
+### Step 2: Map Choreo groups to Asgardeo in the Choreo Console 
 
 !!! note
     
-    Before you map Asgardeo groups to Choreo roles, ensure you meet the following criteria:
+    Before you map Choreo groups to Asgardeo, ensure you meet the following criteria:
```

Comment:
This too should be enterprise IdP and not Asgardeo

Reply:
Done